lib/src/specs/expression.dart

```diff
@@ -303,6 +303,14 @@ abstract class Expression implements Spec {
         addSpace: false,
       );
 
+  /// This expression precenede by the null safe spread operator `?...`.
+  Expression get nullSafeSpread => BinaryExpression._(
+        const LiteralExpression._('?...'),
```

Comment:
I think we want `...?` not `?...` right?

Reply:
Fixed